### PR TITLE
Update send-aws-services-logs-with-the-datadog-kinesis-firehose-desti…

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds a section on how to find logs by AWS Kinesis Firehose ARN in Datadog

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/kinesis-logs-explorer/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md [hunk 1]
```diff
@@ -84,9 +88,16 @@ See the full [Kinesis CloudFormation template][1].
 3. Check the `Subscriptions` column in the [log groups index page][1] to confirm that the new Kinesis stream is now subscribed to your log groups.
 ​
 If you want to push logs directly to the delivery stream without going through a Kinesis data stream, you can subscribe the CloudWatch log groups directly to the Kinesis Firehose Destination by adding the Kinesis Firehose ARN in the `destination-arn` parameter of the subscription filter, as shown in [the AWS Subscription Filters documentation][4] (step 12).
-​
+
+## Searching for AWS Kinesis logs in Datadog
```
Comment (alai97) [inline content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md:92]:
```suggestion
## Search for AWS Kinesis logs in Datadog
```

File: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md [hunk 2]
```diff
@@ -84,9 +88,16 @@ See the full [Kinesis CloudFormation template][1].
...
+
+Once you have set up an Amazon Kinesis delivery stream, logs subscribed to your delivery stream are now available for analysis within Datadog. To populate all logs by ARN:
```
Comment (alai97) [inline content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md:94]:
```suggestion
Once you have set up an Amazon Kinesis delivery stream, you can analyze the logs subscribed to your delivery stream in Datadog. 

To populate all logs by ARN:
```

File: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md [hunk 3]
```diff
@@ -84,9 +88,16 @@ See the full [Kinesis CloudFormation template][1].
...
+
+1. Navigate to [Logs Explorer][5] in Datadog to see all of your subscribed logs.
```
Comment (alai97) [inline content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md:96]:
```suggestion
1. Navigate to the [Logs Explorer][5] in Datadog to see all of your subscribed logs.
```

File: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md [hunk 4]
```diff
@@ -84,9 +88,16 @@ See the full [Kinesis CloudFormation template][1].
...
+2. In the search bar, type `@aws.firehose.arn:"<ARN>"`, replacing `<ARN>` with your Amazon Kinesis Data Firehose ARN, and press enter.
```
Comment (alai97) [inline content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md:97]:
```suggestion
2. In the search bar, type `@aws.firehose.arn:"<ARN>"`, replace `<ARN>` with your Amazon Kinesis Data Firehose ARN, and press **Enter**.
```

File: content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md (hunk 3 above)
Comment (alai97) [reply content/en/logs/guide/send-aws-services-logs-with-the-datadog-kinesis-firehose-destination.md:96]: Should this link send users to the in-app Logs Explorer or the Logs Explorer documentation?

**Review by alai97 — CHANGES_REQUESTED** [review]
Standardize header titles for consistency and a minor grammar nit.

**Comment by sarina-dd** [comment]
@alai97 Thanks for the review! This PR is ready for final review.

**Review by alai97 — APPROVED** [review]

**Pull request merged**